data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
 			choice
 				`	"I'm honored. Thank you."`
-					decline
+					goto onwards
 				`	"Is there anything else I can do to assist your people?"`
-			`	"Not that I am aware of right now," he says, "but we will call on you if a need ever arises."`
-				decline
+			`	"I do not have anything at the moment," he says, "but I am sure other Remnant will have tasks you can help with, now that you have demonstrated your trustworthiness. We have been keeping track of you, and we are impressed with your curiosity, perseverance, and ability to survive where most would be destroyed. All three are valued characteristics among the Remnant, and thus many will welcome your assistance on their projects.`
+			label onwards
+			`	As he talks, he gestures for you to walk with him towards a nearby hall. "While you are welcome to become one of us, we suspect that you are a citizen of the galaxy - wandering hither and yon building coalitions and enmities with species and groups across the breadth of accessible space, unfettered by conventional allegiances. As such, we hope that you will continue to explore and stop by to let us know what you have found. We ask only that you keep the secret with which you are trusted, and bring us word of anything that might influence humanity's long-term survival."`
+			`	You follow the prefect into the hall, and discover that it is one of the many cafeterias that seem to be common in Remnant starports. He picks up two trays from the window and leads you to an empty table.`
+			choice
+				`	"What secret do you want me to keep?`
+					goto secret
+				`	"What constitutes 'things that influence humanity's survival?"`
+			`	He takes a bite before responding. "Basically, anything that might either pose an existential threat to humanity; or anything that we might learn from and put to use to ensure humanity's survival. It might be an invading fleet of automated machines, or a spaceborn plague. But it is just as possible you might find beings that we can work alongside to form strong partnerships, or ancient ruins with old technology that points us towards new discoveries. The possibilities are indeed infinite.`
+				goto feast
+			label secret
+			`	He sips on thick drink before responding. "Quite simply, the fact that we are here. To the best of our knowledge, only the Drak and the Korath know that we exist, and of those only the Korath have raided our worlds. Our intention is to the last bastion of humanity, and we rely on secrecy to achieve that goal." He pauses for a moment to eat, then continues. "The Alphas are a fearsome threat, but they may be the least of humanity's worries in the grand scheme of things. Thus we strive to maintain whatever advantages we can, and one of those is surprise. If no-one knows we are here, we have the advantage over any that might seek to conquer or destroy us."`
```

Comment:
Pat of the point of this statement is hinting at the fact that the Remnant - like the player - are galactic travellers. They know about the the Korath, the Quarg, the Automata, the Hai (and the fact that the Hai seem to have a civil war going on in the North), the Builders, Ka'Het, and the Coalition. And probably the Pug in Deneb, too. More precisely, they have seen everything that one can observe from space of all those. They haven't actually *met* any of them. They have, however, done scans on their ships. The only race that the Remnant *might* not have seen is the Wanderers. They probably have seen them, but that hasn't been decided yet. The Wanderers are the only one (so far) that is isolated enough to even have a chance of being hidden from them.

Giftbringer might be a fearsome Carrier; but it is small potatoes next to a Punisher or Arfecta... And for that matter, while the Alpha's are definitely a threat, they can't match the level of destruction an empire like the Coalition could manage. Who cares if the Alphas are basically elite commandos when one knows about an empire that could probably drop a literal billion soldiers on human space? (Keep in mind, the Remnant know effectively nothing about the Coalition other than what they can see from space - and they would definitely be able to tell that they have a lot of very good military ships and very little to use them against - which historically in humanity tends to mean a nation that is interested in involving itself in the affairs of those around it. 

The actual logic of the "thus" is "There are bigger threats than the Alphas" so "we strive to maintain whatever advantages we can."

Reply:
I think I understand what you're saying here. The suggestion below is a small rewrite: `We strive to maintain whatever advantages we can in case any of those threats proves itself such, and one of our best tools is surprise.` This clarifies a little of what you've explained above and resolves my concerns.
```suggestion
			`	He sips on a thick drink before responding. "Quite simply, the fact that we are here. To the best of our knowledge, only the Drak and the Korath know that we exist, and of those only the Korath have raided our worlds. Our intention is to be the last bastion of humanity, and we rely on secrecy to achieve that goal." He pauses for a moment to eat, then he continues. "The Alphas are a fearsome threat, but they may be the least of humanity's worries in the grand scheme of things. We strive to maintain whatever advantages we can in case any of those threats proves itself such, and one of our best tools is surprise. If no one knows we are here, we have the advantage over any that might seek to conquer or destroy us.`
```